data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/model/AuditErrorMessageTest.java

```diff
@@ -41,53 +42,44 @@
     private JacksonTester<AuditErrorMessage> jsonTester;
 
     /**
-     * Grouped assertion test
-     * Create the object with the builder and then convert to the Json equivalent.
+     * Creates the object with the builder and then convert to the Json equivalent.
      * Takes the JSON Object, deserialises and tests against the original Object
      *
      * @throws IOException throws if the {@link AuditErrorMessage} object cannot be converted to a JsonContent.
      *                     This equates to a failure to serialise or deserialise the string.
      */
     @Test
-    void testGroupedDependantErrorMessageSerialisingAndDeserialising() throws IOException {
-        Context context = new Context().purpose("testContext");
-        Map<String, Object> attributes = new HashMap<>();
-        attributes.put("messagesSent", "23");
+    void testAuditErrorMessageSerialisingAndDeserialising() throws IOException {
+        JsonContent<AuditErrorMessage> auditErrorMessageJsonContent = jsonTester.write(AUDIT_ERROR_MESSAGE);
+        ObjectContent<AuditErrorMessage> auditErrorMessageObjectContent = jsonTester.parse(auditErrorMessageJsonContent.getJson());
+        AuditErrorMessage auditErrorMessageObject = auditErrorMessageObjectContent.getObject();
 
-        AuditErrorMessage auditErrorMessage = AuditErrorMessage.Builder.create()
-                .withUserId("originalUserID")
-                .withResourceId("testResourceId")
-                .withContext(context)
-                .withAttributes(attributes)
-                .withError(new InternalError("Something went wrong!"));
+        assertAll("ObjectComparison",
+                () -> assertThat(auditErrorMessageObject).as("Comparison assertion using the AuditErrorMessage's equals").isEqualTo(AUDIT_ERROR_MESSAGE),
+                () -> assertThat(auditErrorMessageObject).as("Comparison assertion using all of the AuditErrorMessage's components recursively").usingRecursiveComparison().ignoringFieldsOfTypes(Throwable.class).isEqualTo(AUDIT_ERROR_MESSAGE)
```

Comment:
I would prefer to see the assertions over multiple lines as it makes them easier to read and does not involve and side scrolling.
```suggestion
                () -> assertThat(auditErrorMessageObject).as("Comparison assertion using the AuditErrorMessage's equals")
                .isEqualTo(AUDIT_ERROR_MESSAGE),
                () -> assertThat(auditErrorMessageObject).as("Comparison assertion using all of the AuditErrorMessage's components recursively")
                .usingRecursiveComparison().ignoringFieldsOfTypes(Throwable.class)
                .isEqualTo(AUDIT_ERROR_MESSAGE)
```

Reply:
done